habitat_baselines/il/metrics.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import copy
+import json
+from typing import List, Tuple
+
+import numpy as np
+import torch
+
+
+class Metric:
```

Comment:
Shouldn't we subclass the exacting TaskMetrics we already have? If that doesn't work, perhaps we need a new abstractclass Metric type. @mathfac thoughts?

Reply:
Yes, we should call it or our metrics class differently. Call out metric class: HabitatMetric or TaskMetric and then this class call BaselinesMetric.

Reply:
Doc string needed.